src/ansys/hps/client/rms/models.py

```diff
@@ -34,683 +34,836 @@
-        ..., description="Compute resource set details", title="Compute Resource Sets"
+        ..., description='Compute resource set details', title='Compute Resource Sets'
     )
 
 
 class EvaluatorConfiguration(BaseModel):
-    id: Optional[str] = Field(None, description="Unique database ID (read-only).", title="Id")
+    id: Optional[str] = Field(None, description='Unique database ID (read-only).', title='Id')
     evaluator_id: Optional[str] = Field(
-        None, description="ID of the parent evaluator (read-only).", title="Evaluator Id"
+        None, description='ID of the parent evaluator (read-only).', title='Evaluator Id'
     )
     last_modified: Optional[datetime] = Field(
-        None, description="Last modified time", title="Last Modified"
+        None, description='Last modified time', title='Last Modified'
     )
-    working_directory: Optional[str] = Field(None, title="Working Directory")
+    working_directory: Optional[str] = Field(None, title='Working Directory')
     local_file_cache_max_size: Optional[int] = Field(
-        None, description="Maximum allowed cache size in bytes.", title="Local File Cache Max Size"
+        None, description='Maximum allowed cache size in bytes.', title='Local File Cache Max Size'
     )
-    max_num_parallel_tasks: Optional[int] = Field(None, title="Max Num Parallel Tasks")
+    max_num_parallel_tasks: Optional[int] = Field(None, title='Max Num Parallel Tasks')
     task_directory_cleanup: Optional[TaskDirectoryCleanupEnum] = Field(
-        None, title="Task Directory Cleanup"
+        None, title='Task Directory Cleanup'
     )
-    resources: Optional[EvaluatorResources] = {"custom": {}}
-    task_manager_type: Optional[str] = Field(None, title="Task Manager Type")
+    resources: Optional[EvaluatorResources] = {'custom': {}}
+    task_manager_type: Optional[str] = Field(None, title='Task Manager Type')
     loop_interval: Optional[float] = Field(
-        5.0,
+        5,
         description="Number of seconds between each iteration of the evaluator's main loop.",
-        title="Loop Interval",
+        title='Loop Interval',
     )
     local_file_cache: Optional[bool] = Field(
         True,
-        description="Whether to configure a local file cache in the file tool.",
-        title="Local File Cache",
+        description='Whether to configure a local file cache in the file tool.',
+        title='Local File Cache',
     )
     applications: Optional[List[ApplicationInfo]] = Field(
-        [], description="List of available applications.", title="Applications"
+        [], description='List of available applications.', title='Applications'
     )
     project_server_select: Optional[bool] = Field(
         True,
-        description="Get project assignments from the server instead of using the locally set values",
-        title="Project Server Select",
+        description='Get project assignments from the server instead of using the locally set values',
```

Comment:
```suggestion
        description='Get project assignments from the server instead of using the locally set values.',
```
Usually, descriptions starting with verbs are reserved for classes and methods. I'm not sure how to rewrite this so. Would this work: Projects assignments to get from the server rather than using locally set values."